concepts/authorities.py

```diff
@@ -164,3 +165,16 @@ def get_by_namespace(namespace):
 
     return [ manager for manager in authority_managers
                 if manager.namespace == namespace ]
+
+
+def add(instance):
+    """
+    Adding the approved concept to Conceptpower
+
+    Parameters
+    -----------
+    instance: :class: '.Concept'
+    """
+
+    Conceptpower().create(settings.USER_ID, settings.PASSWORD, instance.label, instance.pos,
+                        'VogonWeb Concepts', instance.description, instance.typed)
```

Comment:
Even though we're using the same list for everything, let's declare a variable for the list name so that it's easier to document.